.github/workflows/ci.yml

```
@@ -0,0 +1,83 @@
+    runs-on: ubuntu-latest
+    needs: build
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: latest
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)